src/dataModel/algoModel.js

```diff
@@ -0,0 +1,144 @@
+import { findKey } from "lodash";
+
+let nextAlgoId = 0;
+
+/**
+ * Generates seed algo definition data.
+ * @returns Seed data for algo definitions.
+ */
+function seedAlgos() {
```

Comment:
I wonder if it's worth moving these into a json file in the public directory and just doing an await fetch('./seed.json') here?

Reply:
👍